docs/settings.md

```diff
@@ -1,16 +1,54 @@
 ## Pageserver
 
-### listen_pg_addr
+Pageserver is mainly configured via a `pageserver.toml` config file.
+If there's no such file durinig `init` phase of the server, it crates the file itself, otherwise it does not alter and parses it.
```

Comment:
```suggestion
If there's no such file during `init` phase of the server, it creates the file itself. Without 'init', the file is read.
```
This is a bit confusing, because the `pageserver.toml` file cannot exist when you create a new repository, because the `init` creates the directory.